ray-operator/config/samples/ray-cluster-tpu.yaml

```diff
@@ -0,0 +1,117 @@
+---
+# This template contains a Kuberay cluster using a 2x2x2 TPU v4 PodSlice.
+# To get access to TPU resources, please follow instructions in this doc:
+# https://docs.google.com/document/d/1TRwtfi2pzXbT6We0WQdwNzDdKHruk3u1XeLLJwW38kE/edit#heading=h.8ntu2hqwqvhl
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    app.kubernetes.io/name: kuberay
+    app.kubernetes.io/instance: raycluster-tpu
+  name: raycluster-tpu-kuberay
+  
+spec:
+  headGroupSpec:
+    serviceType: ClusterIP
+    rayStartParams:
+        block: "true"
+        dashboard-host: "0.0.0.0"
+        resources: "\"{\\\"google.com/tpu\\\": 4}\""
+    template:
+      spec:
+        imagePullSecrets:
+          []
+        containers:
+          - volumeMounts:
+            - mountPath: /tmp/ray
+              name: log-volume
+            name: ray-head
+            image: rayproject/ray:2.5.0
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: "8"
+                ephemeral-storage: 20Gi
+                google.com/tpu: "4"
+                memory: 20G
+              requests:
+                cpu: "8"
+                ephemeral-storage: 10Gi
+                google.com/tpu: "4"
+                memory: 20G
+            securityContext:
+              {}
+            env:
+              []
+        volumes:
+          - emptyDir: {}
+            name: log-volume
+        affinity:
+          {}
+        tolerations:
+          []
+        nodeSelector:
+          cloud.google.com/gke-tpu-accelerator: tpu-v4-podslice
+          cloud.google.com/gke-tpu-topology: 2x2x2
+      metadata:
+        annotations:
+          {}
+        labels:
+          cloud.google.com/gke-ray-node-type: head
+          app.kubernetes.io/name: kuberay
+          app.kubernetes.io/instance: raycluster-tpu
+         
+
+  workerGroupSpecs:
+  - rayStartParams:
+        block: "true"
```

Comment:
`block: "true"` will be injected automatically, so we can remove it. https://github.com/ray-project/kuberay/pull/932